packages/dbml-core/src/parse/Parser.js

```diff
@@ -34,6 +34,31 @@ class Parser {
     return postgresParser.parse(str);
   }
 
+  static parseDBMLToJSONv2 (str, DBMLCompiler) {
```

Comment:
Please also recheck if it's possible to rename the class property `this.DBMLCompiler` to `this.dbmlCompiler` as well (both in dbml and in other packages/apps that rely on it). But this is optional, if break too many stuff, we should keep the current name.
```suggestion
  static parseDBMLToJSONv2 (str, dbmlCompiler) {
```

Reply:
I have renamed it nha a